gemmbench/problems.py

```diff
@@ -1009,7 +1038,7 @@ def get_gemm_configs() -> list[tuple[str, GemmConfig]]:
     unet_configs += unet("f16")
     unet_configs += unet("bf16")
 
-    square_configs: list[GemmConfig] = square("f16")
+    square_configs: list[GemmConfig] = square("f16") + square("bf16") + square("i8")
```

Comment:
Co we care about bf16?

Reply:
Some people cared enough to create `V_MFMA_F32_{*}_BF16` :-)

On x86 CPU, bf16 is accelerated by custom instructions which we use; fp16 is not.

On huggingface, the f32 weights many trained models are actually bf16 (the low 16 bits are zero) indicating widespread usage of bf16 in training.